docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -0,0 +1,520 @@
+---
+title: "Model in image"
+date: 2019-02-23T17:32:31-05:00
+weight: 4
+description: "Sample for supplying a WebLogic Deploy Tool (WDT) model that the operator automatically expands into a full domain home during runtime."
+---
+
+### Contents
+
+  - [Introduction](#introduction)
+  - [References](#references)
+  - [Prerequisites for all domain types](#prerequisites-for-all-domain-types)
+  - [Prerequisites for JRF domains](#prerequisites-for-jrf-domains)
+    - [Set Up and Initialize an RCU Database](#set-up-and-initialize-an-rcu-database)
+    - [Increase introspection job timeout](#increase-introspection-job-timeout)
+    - [Set up RCU model attributes, domain resource attributes, and secrets](#set-up-rcu-model-attributes-domain-resource-attributes-and-secrets)
+    - [Reusing or sharing RCU tables](#reusing-or-sharing-rcu-tables)
+  - [Use the WebLogic Image Tool to create an image](#use-the-weblogic-image-tool-to-create-an-image)
+  - [Create and deploy your Kubernetes resources](#create-and-deploy-your-kubernetes-resources)
+  - [Optionally test the sample application](#optionally-test-the-sample-application)
+  - [Optionally access the WebLogic console](#optionally-access-the-weblogic-console)
+  - [Cleanup](#cleanup)
+
+### Introduction
+
+This sample demonstrates:
+
+  - Using the WebLogic Image Tool to create a Docker image that contains a WebLogic install, a WebLogic Deploy Tool (WDT) install, a Java EE servlet application contained within a WDT archive, and a WebLogic domain that's defined using a WDT model file.
+  - Modifying the WDT model that's embedded within the Docker image using a WDT model file that's supplied using a Kubernetes config map.
+  - Defining a `domainHomeSourceType: FromModel` domain resource that references the WDT model image and the WDT config map.
+  - Deploying the model image, domain resource, model config map, and associated secrets that define user names, passwords, and URL values for the model and its domain resource.
+  - Deploying and accessing a Traefik load balancer that redirects HTTP protocol calls to its Java EE servlet application.
+
+Supported domain types:
+
+There are three types of domains supported by Model in Image: a standard `WLS` domain, an Oracle Fusion Middleware Infrastructure Java Required Files (`JRF`) domain, or a `RestrictedJRF` domain.
+
+The `JRF` domain path through the sample includes additional steps for deploying an infrastructure database and initializing the database using the Repository Creation Utility (RCU) tool. `JRF` domains may be  used by Oracle products that layer on top of WebLogic Server such as SOA and OSB. Similarly, `RestrictedJRF` domains may be used by Oracle layered products such as Oracle Communications products.
```

Comment:
extra space "be  used"

Reply:
Fixed for the eagle eye, Ryan.

Reply:
Social distancing is only for us humans, sadly.